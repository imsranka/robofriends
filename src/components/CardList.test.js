// import { shallow, mount, render } from 'enzyme';
import React from "react";
import CardList from './CardList';

import renderer from "react-test-renderer";
 
it("expect to render CardList component", () => {
    const mockRobots = [{
        id:8, 
        username:'yo', 
        name:'lo', 
        email:'ho'
    }]
    const tree = renderer.create(<CardList robots={mockRobots}/>).toJSON();
    expect(tree).toMatchSnapshot();
});
