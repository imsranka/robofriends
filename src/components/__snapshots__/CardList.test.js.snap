// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
<div>
  <div
    className="tc bg-light-green dib br3 pa3 ma2 grow shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/8?size=200x200"
    />
    <div>
      <h2>
        lo
      </h2>
      <p>
        ho
      </p>
    </div>
  </div>
</div>
`;
